static/js/modules/glossary.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+var terms = require('./terms');
+var _ = require('underscore');
+
+var glossaryLink = $('.term'),
+    glossaryIsOpen = false,
+    indexToLowercase,
+    findDefinition,
+    setDefinition,
+    showGlossary,
+    hideGlossary,
+    clearTerm;
+
+// Indexing the terms and then lowercasing the indices
+indexToLowercase = function(arrayOfObjects, index){
+    var key, 
+        newKey,
+        object = _.indexBy(arrayOfObjects, index),
```

Comment:
So I had that, then I made it generic thinking that this function might be useful in other circumstances. But otherwise, yeah, I agree.